docs/user/software/sound/index.md

```diff
@@ -6,3 +6,87 @@ summary: Configuration of sound systems on Solus
 # Sound
 
 Solus comes with Pipewire configured by default.
+
+## Pipewire as a JACK Replacement
+
+Firstly, install `pipewire-jack`
+
+```bash
+sudo eopkg it pipewire-jack
+```
+
+Currently, using `pipewire-jack` as a `JACK` replacement is not as smooth as one would think. One must get their hands dirty a little to get JACK apps to behave appropriately 🛠
+
+### Issues to be fixed
+
+With no configuration pipewire will not be able to switch to the appropriate sample-rate for the project.
+
+![Pipewire Rates Issue](ardour-pipewire-rates-issue.png)
+
+### Configuration
+
+Fortunately both the `pipewire` and `pipewire-jack` packages come with the configs we need.
+
+Open up your terminal of choice and input the commands below
```

Comment:
Add a sentence here explaining what the string of commands will do, like: "The following commands copy the default configurations to your user configuration directory, where you can modify them."